### PR TITLE
Fix mnist download for tpu and hvd tests

### PR DESCRIPTION
**Pull request by trsvchn**
Fixes https://github.com/pytorch/ignite/pull/2061#issuecomment-863858047

Description:

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

**Comment by trsvchn** [comment]
I think that's it

**Review by vfdev-5 — APPROVED** [review]
Thanks @trsvchn !

File: .github/workflows/hvd-tests.yml [hunk 1]
```diff
@@ -60,6 +60,13 @@ jobs:
           pip install horovod
           python setup.py install
 
+      # Download MNIST: https://github.com/pytorch/ignite/issues/1737
+      # to "/tmp" for unit tests
```
Comment (trsvchn) [inline .github/workflows/hvd-tests.yml:64]:
```suggestion
      # to "/tmp" for cpu tests
```

File: .github/workflows/tpu-tests.yml [hunk 2]
```diff
@@ -72,6 +72,13 @@ jobs:
           ln -s libmkl_intel_thread.so.1 libmkl_intel_thread.so
           ln -s libmkl_core.so.1 libmkl_core.so
 
+      # Download MNIST: https://github.com/pytorch/ignite/issues/1737
+      # to "/tmp" for unit tests
```
Comment (trsvchn) [inline .github/workflows/tpu-tests.yml:76]:
```suggestion
      # to "/tmp" for tpu tests
```

**Pull request merged**